docs/book/basic-usage.md

```diff
@@ -0,0 +1,43 @@
+# Basic Usage
+
+In the following example, a flash message is set in an edit action and the rendering is done after a redirect.
+
+## Create a Flash Message
+
+Set a message to the messenger in a controller action, e.g. `module/Album/Controller/AlbumController.php`:
```

Comment:
The start of this sentence doesn't read correctly, to me. How about one of the following?

- Flash (store) a message in the messenger of a controller action
- Flash a message in a controller action
- Store a message in the messenger of a controller action

I feel that one of these, or a further variation, might both more clearly indicate what a Flash Message is and what this section covers. I don't want to dumb things down, but at the same time, want the information to be intuitive and readily-digestible.